utils/gpt.py

```diff
@@ -103,12 +118,14 @@ def _create_payload(self, prompt: str, images: list[str]=[]):
         })
 
         self.past_messages.append(messages)
-
-        return {
+        payload = {
             "model": self.model,
             "system": self.predefined_prompts["system"],
             "messages": self.past_messages,
         }
+        if backend == "Claude":
+            payload["max_tokens"] = 3000
```

Comment:
A max token value must be specified in the payload for the official APIs. 
Feel free to suggest a different value if 3k does not sound correct.